apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultInfo/Messages.tsx

```diff
@@ -21,7 +21,13 @@ export function EvaluationResultMessages({
     return [...(providerLog.messages as Message[]), responseMessage]
   }, [providerLog])
 
-  if (!providerLog) return null
+  if (!providerLog) {
+    return (
+      <Text.H5 color='foregroundMuted' centered>
+        There are no messages generated for this log
+      </Text.H5>
+    )
+  }
 
```

Comment:
Sometimes, the messages take some time to load, especially when a batch evaluation is currently being run. We should not display a message stating that a log does not exist when it is only still loading.

Not sure if this is the case here, but just something to take into account!

Reply:
People would have to click on the row and on message tab super quickly. I'll leave this way for now. We can always pass `isLoading` from store

Reply:
Not critical. Not doing here sorry